### PR TITLE
Feature/list view list view changes

### PR DESCRIPTION
**Pull request by sreeragmsurendran**

### CODE REVIEW

File: src/components/Listview/ListView.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import ListViewItem from '../Listviewitem/ListViewItem';
+
+ const ListView = () => {
+    let ListItems =[{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"},{item:"jkshjkhdhksd"}]
```
Comment (Evrz) [inline src/components/Listview/ListView.jsx:5]: Add TODO: remove static data when api is integrated or something..

File: src/components/Listviewitem/ListViewItem.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const ListViewItem = (props) => {
+    console.log(props, "asdmjk")
```
Comment (Evrz) [inline src/components/Listviewitem/ListViewItem.jsx:4]: remove console.log

File: src/components/home/index.jsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import DefaultLayout from "../layouts/DefaultLayout";
+import ListView from '../Listview/ListView';
+import DefaultLayout from '../layouts/DefaultLayout';
 
 const Home = ({ user, todos }) => {
```
Comment (Evrz) [inline src/components/home/index.jsx:4]: remove user, todos

File: src/components/Listview/ListView.jsx (hunk 1 above)
Comment (sreeragmsurendran) [reply src/components/Listview/ListView.jsx:6]: done

**Pull request merged**